### PR TITLE
fix(wallet): add index on transfers(blk_number, loaded)

### PR DESCRIPTION
**Pull request by stefandunca**
Improve the performance of queries for
- transfer.GetTransfersInRange
- transfer.GetTransfersByAddress
- transfer.GetTransfersByAddressAndBlock
- transfer.GetTransfers
- transfer.GetPreloadedTransactions

For 16952 entries worst case scenario tested with `sqlcipher`:
- Before: Run Time: real 0.897 user 0.728139 sys 0.166714
- After:  Run Time: real 0.001 user 0.000437 sys 0.000189

A single composite index (with the default one) might work better though

status-desktop PR [#10877](https://github.com/status-im/status-desktop/pull/10877)

### CODE REVIEW

**Comment by ghost** [comment]
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://status.im/docs/))?
- [ ] Have you tested changes with mobile?
- [ ] Have you tested changes with desktop?

**Review by alaibe — APPROVED** [review]

File: appdatabase/migrationsprevnodecfg/bindata.go
Comment (dlipicar) [inline appdatabase/migrationsprevnodecfg/bindata.go:1]: is this file desired?

Comment (stefandunca) [reply appdatabase/migrationsprevnodecfg/bindata.go:1]: It is refreshed when I run `go generate ./appdatabase/...` any other way to only refresh `migrations` only?

Comment (stefandunca) [reply appdatabase/migrationsprevnodecfg/bindata.go:1]: I rebased and discarded the `appdatabase/migrationsprevnodecfg/bindata.go`

**Pull request merged**